lib/osf-components/addon/components/file-browser/file-rename-modal/component.ts

```diff
@@ -27,12 +27,16 @@ export default class FileRenameModal extends Component<Args> {
     }
 
     get isValid() {
-        return this.newFileName !== this.originalFileName;
+        return this.newFileName !== this.originalFileName && this.newFileName !== '' && this.newFileName !== null;
     }
 
     @action
     resetFileNameValue() {
-        this.newFileName =  this.originalFileName;
+        return this.newFileName = this.originalFileName;
+    }
+
+    updateOriginalName(newName: string) {
+        return this.originalFileName = newName;
     }
```

Comment:
Since we no longer need this, plz remove.

Reply:
Of course, this is now removed